Frigate_Camera_Notifications/Beta.yaml

```diff
@@ -104,6 +104,26 @@ blueprint:
 
         Note: If the group contains both mobile devices and TVs, the TV will not display the snapshot unless "TV notifications" is set to true. However, this will stop Android phones from receiving thumbnails.
       default: ""
+    notify_telegram_chat_id:
+      name: Telegram Chat ID (Optional)
+      description: |
+        The chat ID to send telegram notifications to.
+        If set, this will override individual devices and groups above. Note: TV (far below) must be set to false.
+        Base URL must start with https since telegram URL buttons only work with https.
+      default: ""
+    frigate_url:
+      name: Frigate API URL (Optional)
+      description: |
+        The Frigate API URL for downloading files for telegram integration.
+        Required if Telegram Chat ID is set and bearer token is unset.
+      default: ""
+    bearer_token:
+      name: Bearer token (Optional)
+      description: |
+        Bearer token is used instead of accessing Frigate API via unsecure ports.
+        Required if Telegram Chat ID is set and Frigate API URL is unset.
+        Bearer token should be generated as HS256 JWT with Frigate JWT secret.
+      default: ""
```

Comment:
I dont really know telegram well. do we need this extra authentication to work even though you are using the same url we alreayd use for every other notification.

Reply:
It's not really about telegram, it's about frigated having auth.

We access those urls with devices that are authenticated with username and password
That is accessed at 8971 port Web UI/API
port 5000 is unauthenticated access (which we can use for frigate_url/api_url)

Any other client (like telegram integration, which is a python script that's run inside home assistant)
needs to retrieve those images. A way to retrieve those images is either over port 5000 (without auth) or with bearer token at port 8971.

https://docs.frigate.video/configuration/authentication/

Reply:
But the http api from the frigate integration that we use for everything else is no good for telegram? 

https://docs.frigate.video/integrations/api

Reply:
HTTP API used for HA frigate integration is unauthenticated.
If I understood correctly, one that is recommended to be used for `base_url` is from port 8971, which is Web UI + API. If we use bearer_token we can use this API since it's authenticated.
If we want to avoid bearer token generation, we can use one from port 5000 which we want to set at `frigate_url`.

This second approach is fine if you have that port firewalled so it's only accessible from HA instance

Reply:
I misspoke, we use th enorification API currently for all notifications. This means frigate doesn't have to be exposed to the world which i think is how most people have it set up. 

https://docs.frigate.video/integrations/home-assistant

Reply:
I did not know that was a thing. In that case my API URL / Bearer token implementation is not needed.
I will try to adjust accordingly